resend.yaml

```diff
@@ -1,93 +1,548 @@
+          items:
+            type: string
+            description: The email addresses of the recipients.
+          example: ['delivered@resend.dev']
+        from:
+          type: string
+          description: The email address of the sender.
+          example: 'Acme <onboarding@resend.dev>'
+        created_at:
+          type: string
+          format: date-time
+          description: The date and time the email was created.
+          example: '2023-04-03T22:13:42.674981+00:00'
+        subject:
+          type: string
+          description: The subject line of the email.
+          example: 'Hello World'
+        html:
+          type: string
+          description: The HTML body of the email.
+          example: 'Congrats on sending your <strong>first email</strong>!'
+        text:
+          type: string
+          description: The plain text body of the email.
+        bcc:
+          type: array
+          items:
+            type: string
+          description: The email addresses of the blind carbon copy recipients.
+        cc:
+          type: array
+          items:
+            type: string
+          description: The email addresses of the carbon copy recipients.
+        reply_to:
+          type: array
+          items:
+            type: string
+          description: The email addresses to which replies should be sent.
+        last_event:
+          type: string
+          description: The status of the email.
+          example: 'delivered'
+    CreateBatchEmailsResponse:
+      type: object
+      properties:
+        data:
+          type: array
+          items:
+            type: object
+            properties:
+              id:
+                type: string
+                description: The ID of the sent email.
+    CreateDomainRequest:
+      type: object
+      required:
+        - name
+      properties:
+        name:
+          type: string
+          description: The name of the domain you want to create.
+        region:
+          type: string
+          enum:
+            - us-east-1
+            - eu-west-1
+            - sa-east-1
+          default: us-east-1
+          description: The region where emails will be sent from. Possible values are us-east-1' | 'eu-west-1' | 'sa-east-1
+    CreateDomainResponse:
+      type: object
+      properties:
+        id:
+          type: string
+          description: The ID of the domain.
+        name:
+          type: string
+          description: The name of the domain.
+        created_at:
+          type: string
+          format: date-time
+          description: The date and time the domain was created.
+        status:
+          type: string
+          description: The status of the domain.
+        records:
+          type: array
+          items:
+            $ref: '#/components/schemas/DomainRecord'
+        region:
+          type: string
+          description: The region where the domain is hosted.
+        dnsProvider:
```

Comment:
I believe `dnsProvider` is outdated, can you double check?

Reply:
I've confirmed that dnsProvider doesn't exist on the Node SDK.